### ISSUE TITLE
Include installation instructions

### ISSUE THREAD

**Issue by dpshelio** [UCL-ARC/git-novice-vscode#2]
From previous workshops, this is what I had:

- Account on Github
- [Download VS Code](https://code.visualstudio.com/download) (or [vscodium](https://vscodium.com/) - untested if gh connection works too)
- Install git
  - Windows?
     - Follow the [Bash shell installation instructions](https://carpentries.github.io/workshop-template/#the-bash-shell) ~~Install [GitHub Desktop](https://desktop.github.com/)~~ (that doesn't help)
  - Mac?
    - Open a terminal window
    - Type:
      ```
      /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
      ```
    - Follow the installation prompts
    - Once it's installed, run: `brew install git` to install git.

**Comment by dtwwwc1e** [comment]
I think the install instructions are fine - install vs code, then install git. I don't have a "clean pc" to test but I can try a VM. We need to enable Bash in vs code? Or that config step is in the taught session?

**Comment by dtwwwc1e** [comment]
I will have VM to test on Monday.

**Comment by dtwwwc1e** [comment]
> I will have VM to test on Monday.

Tested out on a VM Windows, all ok for a fresh install. VSCode run fine, Git acting as expected. Bash allows `git gui` too which I was not expecting. So install instructions all good.

I have not tested out vscodium, or access to a Mac. I will try the first one when I can get a VM again. Can't promise the Mac.

**Comment by dtwwwc1e** [comment]
Installed vscodium on windows and did some local staging, and commit. Working ok.

**Comment by dpshelio** [comment]
Did the connection to github worked with vscodium? i.e., being able to push without ssh keys but by pressing push?

**Comment by dtwwwc1e** [comment]
I didn't test vscodoum - github. Will test it today or tomorrow

**Comment by dtwwwc1e** [comment]
Yes, push and pull / fetch works - I already have ssh keys set up (and am using ucl laptop) but I used the HTTPS connection (https://github.com/dtwwwc1e/repo32.git). Push went fine, following various confirmation steps; and fetch / pull worked too.
